src/servicebus/azext_servicebus/_params.py

```diff
@@ -0,0 +1,357 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands.parameters import (tags_type, get_enum_type, resource_group_name_type, name_type)
+
+from azext_servicebus._validators import _validate_auto_delete_on_idle, \
+    _validate_duplicate_detection_history_time_window, _validate_default_message_time_to_live, _validate_lock_duration
+
+
+# pylint: disable=line-too-long
+def load_arguments_namespace(self, _):
+    with self.argument_context('servicebus namespace exists') as c:
+        c.argument('namespace_name', options_list=['--name'], help='name of the Namespace')
```

Comment:
For all `--name` options, also add `-n` to the list so short options work. They are very common in the CLI.

Reply:
Resolved